webui/react/src/pages/Cluster/ClusterHistoricalUsage.tsx

```diff
@@ -25,17 +26,22 @@ export enum GroupBy {
   Month = 'month',
 }
 
+const GROUP_BY_KEY = 'group-by';
+const STORAGE_PATH = 'cluster/historical-usage';
```

Comment:
**super nit:**
```
const STORAGE_PATH = 'cluster/historical-usage';
const STORAGE_GROUP_BY_KEY = 'group-by';
```

Reply:
great to have somebody caring about the little stuff :)